generators/repo/src/commands/generate.ts

```diff
@@ -213,15 +213,16 @@ export class GenerateCommand implements RepomanCommand {
         console.info(chalk.white(`Cloning repo for remote...`));
         await repo.clone(remote.name, remote.url);
 
-        const isEmptyRepo = await repo.isEmptyRepo(remote.name);
+        const defaultBranchExists = await repo.remoteBranchExists(remote.name, defaultBranch);
 
-        if (isEmptyRepo) {
-            console.warn(chalk.yellowBright(`Remote is empty!`));
+        if (!defaultBranchExists) {
+            console.warn(chalk.yellowBright(`Remote does not have branch ${chalk.cyan(defaultBranch)}`));
```

Comment:
Did you confirm this change still works on an empty repo without any branches?  This use case would be for a branch new repo that doesn't have any commits pushed yet.

Reply:
```
...
************************************************************
************ Remote: djurek-empty-template-repo ************
************************************************************
Remote: https://github.com/danieljurek/test-empty-template-repo.git
Host: github.com
Org: danieljurek
Repo: test-empty-template-repo

Cloning repo for remote...
Remote does not have branch main
Creating default branch main...
Branch 'djurek/test-pr' doesn't exist on remote
Creating new branch: djurek/test-pr
Staging files from generated output

Found the following 91 change(s)... 
- A  .azdo/pipelines/README.md      
- A  .azdo/pipelines/azure-dev.yml  
- A  .devcontainer/Dockerfile       
- A  .devcontainer/devcontainer.json
- A  .gitattributes
... <files> 
- A  tests/playwright.config.ts
- A  tests/todo.spec.ts
Committing 91 change(s) to remote with message: Synchronize repo from Repoman

Pushing changes to djurek/test-pr...

Changes are available @ https://github.com/danieljurek/test-empty-template-repo/tree/djurek/test-pr
Compare @ https://github.com/danieljurek/test-empty-template-repo/compare/main...djurek/test-pr
```